site/_posts/2016-10-03-jekyll-3-3-0-released.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: news_item
+title: 'Jekyll 3.3 is here with better theme support, new URL filters, and tons more'
+date: 2016-10-03 17:08:38 -0700
+author: parkr
+version: 3.3.0
+categories: [release]
+---
+
+We have tons of new features for you in our latest release of Jekyll. Three
+key things you might want to give a whirl:
+
+### 1. Themes can now ship static & dynamic assets in an `/assets` directory
+
+We're really stoked about this one. In Jekyll 3.2, we shipped the ability
+to use a theme that was packaged as a gem. Due to security necessities and
+ease-of-use concerns, this initial ship only included support for includes,
+layouts, and sass partials. A theme couldn't write any CSS, JavaScript, or
+content to your site.
+
+In an effort to make theme management a bit easier, any files you put into
+`/assets` in your theme will be read in as though they were part of the
+user's site. This means you can ship SCSS and CoffeeScript, images and
+webfonts, JSON and other data. Same rules apply here as in a Jekyll site:
+if it has YAML front matter, it will be converted and renderd. No YAML
+front matter, and it will simply be copied over like a static asset. Neat,
+huh?
+
+See our [documentation on the subject]({{ "/docs/themes/#assets" | relative_url }})
+for more info.
+
+### 2. `relative_url` and `absolute_url` filters
+
+Want a clean way to prepend the `baseurl` or `url` in your config? These
+new filters have you covered. When working locally, if you set your
+`baseurl` to match your deployment environment, say `baseurl: "/myproject"`,
```

Comment:
Do we want to use the opportunity to explain how `baseurl` *should* be used? I can see lots of users using this filter and setting their baseurl to `/` as a result.